addons/attributes/functions/fnc_gui_combo.sqf

```diff
@@ -0,0 +1,56 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Initializes the `combo` attribute control type.
+ *
+ * Arguments:
+ * 0: Controls Group <CONTROL>
+ * 1: Entity <OBJECT|GROUP|ARRAY|STRING>
+ * 2: Default Value <STRING>
+ * 3: Value Info <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [CONTROL, _entity, "", []] call zen_attributes_fnc_gui_combo
+ *
+ * Public: No
+ */
+
+params ["_controlsGroup", "_entity", "_defaultValue", "_valueInfo"];
+
+private _ctrlCombo = _controlsGroup controlsGroupCtrl IDC_ATTRIBUTE_COMBO;
+_ctrlCombo setVariable [QGVAR(params), [_controlsGroup]];
+
+{
+    _x params ["_value", "_displayName", ["_tooltip", "", [""]], ["_pictureArg", "", ["", []]]];
+    _pictureArg params [["_picture", "", [""]], ["_pictureColor", [1, 1, 1, 1], [[]], [4]]];
```

Comment:
Push one line down

Reply:
I wouldn't say so, both are params and it is cleaner to keep those together